recipes/pynvjitlink/conda_build_config.yaml

```diff
@@ -0,0 +1,10 @@
+c_compiler_version:
+  - 11
+cxx_compiler_version:
+  - 11
+cuda_compiler:
+  - cuda-nvcc
+cuda_compiler_version:
+  - 12.2
+cuda_compiler_version_min:
+  - 12.2
```

Comment:
We always want to use the latest available CUDA version. Maybe that means the pinnings should just be "12" instead of "12.x". Then conda will automatically choose the latest minor version.